### DISCUSSION TITLE
error when analysing image to extract data from tables

### DISCUSSION THREAD

**Discussion by bilel910** [deepdoctection/deepdoctection#314] [Q&A] [answered]
![image](https://github.com/deepdoctection/deepdoctection/assets/83314544/50f5c2b5-8a98-488b-b3f8-04f6c969e005)

**Comment by JaMe76** [comment, marked as answer]
Seems that image_path is pointing to an image file. Make sure that the path is pointing to either a directory of image files or to a single ( maybe multi-page) .pdf document.
